doc-content/enterprise-only/openshift/rhdm73-optaweb-employee-rostering-ref.adoc

```diff
@@ -0,0 +1,282 @@
+|`RHDM_PASSWORD` | Optaweb employee rostering PostgreSQL database password | `${OPTAWEB_POSTGRESQL_PWD}`
+|`RHDM_SERVICE_HOST` | -- | `${APPLICATION_NAME}-postgresql`
+|`RHDM_SERVICE_PORT` | -- | 5432
+|`OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_DATASOURCE` | Optaweb employee rostering persistence datasource (Sets the org.optaweb.employeerostering.persistence.datasource system property) | `${OPTAWEB_PERSISTENCE_DS}`
+|`OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_DIALECT` | -- | org.hibernate.dialect.PostgreSQLDialect
+|OPTAWEB_EMPLOYEE_ROSTERING_PERSISTENCE_HBM2DDL_AUTO | Optaweb employee rostering persistence hbm2ddl auto (Sets the org.optaweb.employeerostering.persistence.hbm2ddl.auto system property) | `${OPTAWEB_PERSISTENCE_HBM2DDL_AUTO}`
+|`OPTAWEB_GENERATOR_ZONE_ID` | The timezone id used by the OptaWeb Generator (Sets the optaweb.generator.zoneId system property; defaults to local timezone) | `${OPTAWEB_GENERATOR_ZONE_ID}`
+|`RHDM_JTA` | -- | true
+|`HTTPS_KEYSTORE_DIR` | -- | `/etc/optaweb-employee-rostering-secret-volume`
+|`HTTPS_KEYSTORE` | The name of the keystore file within the secret | `${OPTAWEB_HTTPS_KEYSTORE}`
+|`HTTPS_NAME` | The name associated with the server certificate | `${OPTAWEB_HTTPS_NAME}`
+|`HTTPS_PASSWORD` | The password for the keystore and certificate | `${OPTAWEB_HTTPS_PASSWORD}`
+|`JGROUPS_PING_PROTOCOL` | -- | openshift.DNS_PING
+|`OPENSHIFT_DNS_PING_SERVICE_NAME` | -- | `${APPLICATION_NAME}-optaweb-employee-rostering-ping`
+|`OPENSHIFT_DNS_PING_SERVICE_PORT` | -- | 8888
+.4+| `${APPLICATION_NAME}-postgresql`
+|`POSTGRESQL_USER` | Optaweb employee rostering PostgreSQL database username | `${OPTAWEB_POSTGRESQL_USER}`
+|`POSTGRESQL_PASSWORD` | Optaweb employee rostering PostgreSQL database password | `${OPTAWEB_POSTGRESQL_PWD}`
+|`POSTGRESQL_DATABASE` | Optaweb employee rostering PostgreSQL database name | `${OPTAWEB_POSTGRESQL_DB}`
+|`POSTGRESQL_MAX_PREPARED_TRANSACTIONS` | Allows the PostgreSQL to handle XA transactions. | `${POSTGRESQL_MAX_PREPARED_TRANSACTIONS}`
+|=======================================================================
+
+
+
+=====  Volumes
+
+|=============
+|Deployment |Name  | mountPath | Purpose | readOnly 
+
+|`${APPLICATION_NAME}-optaweb-employee-rostering` | optaweb-employee-rostering-keystore-volume | `/etc/optaweb-employee-rostering-secret-volume` | ssl certs | True
+|`${APPLICATION_NAME}-postgresql` | `${APPLICATION_NAME}-postgresql-pvol` | `/var/lib/pgsql/data` | postgresql | false
+|=============
+
+
+=== External Dependencies
+
+
+==== Volume Claims
+
+A `PersistentVolume` object is a storage resource in an OpenShift cluster. Storage is provisioned by an administrator
+by creating `PersistentVolume` objects from sources such as GCE Persistent Disks, AWS Elastic Block Stores (EBS), and NFS mounts.
+Refer to the https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/developer_guide/dev-guide-persistent-volumes[Openshift documentation] for
```

Comment:
Same style comment.

Reply:
as above

Reply:
Not commenting on autogen content from this point as I can't really change it reliably. I will instead add a pointer to this PR from the general restructure task so I can work with Dev to handle it all.